test/tsd.ts

```diff
@@ -106,6 +106,25 @@ function testDataQueryBuilderClass() {
     const query: Backendless.DataQueryValueI = dataQuery.build();
 }
 
+function testDataJSONUpdateBuilder() {
+    const jsonUpdateBuilder = Backendless.JSONUpdateBuilder
+
+    const argKey: string = '$.number'
+    const argValue: number = 11
+
+    let jsonPayload: object = jsonUpdateBuilder.SET().addArgument(argKey, argValue).create()
+
+    jsonPayload = jsonUpdateBuilder.INSERT().addArgument(argKey, argValue).create()
```

Comment:
also need to add checking on the instance

```
jsonUpdateBuilder = jsonUpdateBuilder.INSERT().addArgument(argKey, argValue)
```

Reply:
+